harness/determined/estimator/_estimator_trial.py

```diff
@@ -748,9 +748,9 @@ def _init_paths(self) -> None:
             logging.debug(f"Load path set to {self.estimator_dir}.")
 
             # Load WorkloadSequencer state.
-            wlsq_path = os.path.join(load_path, "workload_sequencer.pkl")
-            if self.wlsq is not None and os.path.exists(wlsq_path):
-                with open(wlsq_path, "rb") as f:
+            wlsq_path = load_path.join("workload_sequencer.pkl")
```

Comment:
I don't think `pathlib.Path` has `.join`? prob should be `joinpath` or `/`?
```suggestion
            wlsq_path = load_path / "workload_sequencer.pkl"
```

Reply:
good catch.  I'm going to see if I can't figure out any why this wasn't caught by the linter, and look for similar errors.  e2e tests indicate there are a couple more.

Reply:
It's not caught because apparently TrialController.context has always been an `Any`.

I experimented with fixing it but it looks, surprisingly, very involved.  So I don't have time to do it today